package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "install": "node install",
     "uninstall": "node uninstall",
-    "test": "jshint lib/ *.js && jscs lib/ *.js && nodeunit test.js"
+    "test": "standard lib/*.js && nodeunit test.js"
```

Comment:
Do we still need the argument passed to `standard`? Let's try if calling just `standard` also works (and fails properly).